source/uwp/Renderer/lib/AdaptiveTimeInputRenderer.cpp

```diff
@@ -22,12 +22,54 @@ namespace AdaptiveNamespace
     }
     CATCH_RETURN;
 
-    HRESULT AdaptiveTimeInputRenderer::Render(_In_ IAdaptiveCardElement* cardElement,
+    HRESULT AdaptiveTimeInputRenderer::Render(_In_ IAdaptiveCardElement* adaptiveCardElement,
                                               _In_ IAdaptiveRenderContext* renderContext,
-                                              _In_ IAdaptiveRenderArgs* renderArgs,
-                                              _COM_Outptr_ ABI::Windows::UI::Xaml::IUIElement** result) noexcept try
+                                              _In_ IAdaptiveRenderArgs* /*renderArgs*/,
+                                              _COM_Outptr_ IUIElement** timeInputControl) noexcept try
     {
-        return XamlBuilder::BuildTimeInput(cardElement, renderContext, renderArgs, result);
+        ComPtr<IAdaptiveHostConfig> hostConfig;
+        RETURN_IF_FAILED(renderContext->get_HostConfig(&hostConfig));
+        if (!XamlHelpers::SupportsInteractivity(hostConfig.Get()))
+        {
+            renderContext->AddWarning(
+                ABI::AdaptiveNamespace::WarningStatusCode::InteractivityNotSupported,
+                HStringReference(L"Time Input was stripped from card because interactivity is not supported").Get());
+            return S_OK;
+        }
+
+        ComPtr<ITimePicker> timePicker =
+            XamlHelpers::CreateXamlClass<ITimePicker>(HStringReference(RuntimeClass_Windows_UI_Xaml_Controls_TimePicker));
+
+        // Make the picker stretch full width
+        ComPtr<IFrameworkElement> timePickerAsFrameworkElement;
+        RETURN_IF_FAILED(timePicker.As(&timePickerAsFrameworkElement));
+        RETURN_IF_FAILED(timePickerAsFrameworkElement->put_HorizontalAlignment(ABI::Windows::UI::Xaml::HorizontalAlignment_Stretch));
+        RETURN_IF_FAILED(timePickerAsFrameworkElement->put_VerticalAlignment(ABI::Windows::UI::Xaml::VerticalAlignment_Top));
+
+        RETURN_IF_FAILED(XamlHelpers::SetStyleFromResourceDictionary(renderContext,
+                                                                     L"Adaptive.Input.Time",
+                                                                     timePickerAsFrameworkElement.Get()));
+
+        ComPtr<IAdaptiveCardElement> cardElement(adaptiveCardElement);
+        ComPtr<IAdaptiveTimeInput> adaptiveTimeInput;
+        THROW_IF_FAILED(cardElement.As(&adaptiveTimeInput));
```

Comment:
>THROW_IF_FAILED [](start = 8, length = 15)

I assume this is copy paste, but there are some THROW_IF_FAILEDs in here that should probably be updated to RETURN_IF_FAILED